### PR TITLE
Fix for SLING-7815

### PR DESCRIPTION
**Pull request by csaboka**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -54,6 +54,7 @@
             <plugin>
                 <groupId>org.apache.sling</groupId>
                 <artifactId>maven-sling-plugin</artifactId>
+                <version>2.3.2</version>
```
Comment (kwin) [inline pom.xml]: Please inherit the version from the parent

File: pom.xml [hunk 2]
```diff
@@ -176,7 +177,7 @@
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.api</artifactId>
-            <version>2.16.4</version>
+            <version>2.18.2</version>
```
Comment (kwin) [inline pom.xml]: Do you use updated API really?

File: pom.xml (hunk 1 above)
Comment (csaboka) [reply pom.xml]: Thank you for spotting this. Fixed.

File: pom.xml (hunk 2 above)
Comment (csaboka) [reply pom.xml]: Yes, I do. ResourceProvider.AUTH_CLONE was introduced in 2.18.2, specifically to support the original change this PR is trying to fix.

**Comment by csaboka** [comment]
I've found an issue that made my unit test fail when run as part of the whole suite, but not on its own. I hope it's OK that I've pushed a workaround for it, so things keep working until I can get around to reporting this issue to the Oak project.

**Comment by csaboka** [comment]
Is there anything I can do to help this getting merged? I think I answered all the concerns raised, but maybe I'm missing something.

**Comment by rombert** [comment]
@kwin - do you still have comments on this PR?

**Comment by csaboka** [comment]
@kwin , can you please accept this, or let me know what needs to be changed? This ticket has been in limbo for quite a long time. I'd be happy to fix anything that prevents this from being merged, but at the moment there are no clear issues as far as I know.

**Comment by kwin** [comment]
@csaboka Looks good to me now.
@rombert Can you take up running ITs with it and the merge in case of success?

**Comment by rombert** [comment]
All ITs passed, thanks for your patience @csaboka !

**Pull request merged**